spec/categories_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './lib/models/dish.rb'
+
+describe Dish do
+
+  it { is_expected.to have_property :category }
+
+
+end
```

Comment:
Instead of making a whole file for this, you can add this line to the `dish_spec`.